src/main/kotlin/graphics/scenery/backends/Display.kt

```diff
@@ -116,4 +117,12 @@ interface Display {
             null
         }
     }
+
+    /**
+     * Fades the view on the HMD to the specified color.
+     *
+     * <p>The fade will take {@code seconds}, and the color values are between 0.0 and 1.0. This color is faded on top of the scene based on the alpha
```

Comment:
This is Kdoc, not Javadoc - you can make your life much easier and use Markdown syntax :+1:

Reply:
I'd also switch the seconds and color paramters, and make color a Vector4f that includes alpha. What background does is missing in the doc, though.